src/broker/cas_cgw.c

```diff
@@ -554,7 +559,7 @@ cgw_cur_tuple (T_NET_BUF * net_buf, T_COL_BINDER * first_col_binding, int cursor
 }
 
 int
-cgw_get_col_info (SQLHSTMT hstmt, T_NET_BUF * net_buf, int col_num, T_ODBC_COL_INFO * col_info)
+cgw_get_col_info (SQLHSTMT hstmt, int col_num, T_ODBC_COL_INFO * col_info)
 {
```

Comment:
trivial (in line# 570 ~ 577): It seems that memset and consecutive assignments are duplicates.

```
  memset (col_info, 0x0, sizeof (T_ODBC_COL_INFO));

  col_info->is_unique_key = 0;
  col_info->is_primary_key = 0;
  col_info->is_reverse_index = 0;
  col_info->is_reverse_unique = 0;
  col_info->is_foreign_key = 0;
  col_info->is_shared = 0;
```